Source/Scene/LabelCollection.js

```diff
@@ -339,13 +437,21 @@ define([
         this._scene = options.scene;
 
         this._textureAtlas = undefined;
+        this.__backgroundTextureAtlas = undefined;
+        this._whitePixelIndex = undefined;
+
+        this._backgroundBillboardCollection = new BillboardCollection({
+            scene : this._scene
+        });
+        this._backgroundBillboardCollection.destroyTextureAtlas = false;
 
         this._billboardCollection = new BillboardCollection({
             scene : this._scene
         });
         this._billboardCollection.destroyTextureAtlas = false;
 
         this._spareBillboards = [];
+        this._spareBackgroundBillboards = [];
```

Comment:
Are you sure about needing this?  I don't think we optimize for cases like this anywhere else in Cesium.